src/main/proto/ga4gh/genotype_phenotype_service.proto

```diff
@@ -80,28 +89,34 @@ message EvidenceQuery {
     // ECO or OBI is recommended
     OntologyTerm evidenceType = 1;
 
-    string description = 2;  /*regex*/
+    // regex
+    string description = 2;
 
-    repeated ExternalIdentifier external_identifiers = 3; /* new field */
+    // new field
+    repeated ExternalIdentifier external_identifiers = 3;
 }
 
 message SearchPhenotypesRequest {
     // The `PhenotypeAssociationSet` to search. Mandatory
     string phenotype_association_set_id = 1;
 
+    // Phenotype ID
     string id = 2;
 
-    string description = 3; /* The system should support regular expression matches */
+    // The system should support regular expression matches
```

Comment:
@david4096  We have consistently gotten feedback from the g2p task team that some type of wildcard is desirable. The auxiliary verb "should" was used deliberately as opposed to "must".  This should give, I hope, enough room. 
Per RFC 2119 [should] "mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course."